src/main/java/core/basesyntax/App.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Figure;
+
+public class App {
+    public static void main(String[] args) {
+        final FigureSupplier figureSupplier = new FigureSupplier();
+        final int figuresArraySize = 6;
```

Comment:
to create constant means:
```suggestion
private static final int FIGURES_ARRAY_SIZE = 6;

    public static void main(String[] args) {
        final FigureSupplier figureSupplier = new FigureSupplier();
```